.gitignore

```diff
@@ -55,3 +55,4 @@ var/
 ._*
 # MacOS Desktop Services Store
 .DS_Store
+.venv
```

Comment:
Only needed because I'm working from GitHub Codespaces. I can remove once PR is otherwise done, if needed.